src/components/interface/Accordion/Accordion.js

```diff
@@ -52,7 +52,7 @@ const Accordion = ({
     return cloneElement(child, {
       id,
       key: id,
-      onClick: expand,
+      onClickInternal: expand,
```

Comment:
`onClickItem` ?

Reply:
onClickItem will be confusing with the onClick on the item...

Reply:
tried another way :)